src/lib/expansion/expansion.md

```diff
@@ -0,0 +1,81 @@
+`<md-expansion-panel>` provides a way to show and hide lightweight content, by collapsing and expanding a view with a nice animation.
```

Comment:
```markdown
`<md-expansion-panel>` provides a expandable details-summary view.
```

Statements like "with a nice animation" are subjective. The docs should aim to be more "just the facts". 

The word "lightweight" from the spec doesn't really have any bearing on the implementation. The designers used this word in the spec to communicate that people shouldn't dump an entire application inside an expansion panel.